content/setup/_index.md

```diff
@@ -25,4 +25,5 @@ Below are brief descriptions of the roles each of the machines play in the syste
 
 Now, follow the steps for your environment:
 
-- [Packet](/setup/packet)
+- [Local with Vagrant](/setup/local-with-vagrant)
+- [Packet with Terraform](/setup/packet-with-terraform)
```

Comment:
Packet is still easier IMHO - I wouldn't want to install Vagrant and etc just to try this. So it may be better having it listed first as it's also 'fool proof"

Reply:
One thing about the Packet flow is it takes a fairly significant investment in time (waiting for hardware to provision and boot etc.) and dollars ($1/h minimum commit.) @mrmrcoleman @gianarb and i were talking and we're inclined two put Vagrant top because it is cheapest and has a much shorter cycle time when iterating on workflows.

Reply:
In the past Jacob had found a way to generate a discount code / burner code for a few hundred bucks. This was also used at CES I believe with Cody. Any thoughts on that?

Reply:
Why do we have to choose when you can have both? :D and even more, in the process of working on the Vagrant setup we changed and learned a lot of things on the setup.sh and the workflow is much more reliable.

The fact that the title is "Local with Vagrant" is because we know that a lot of people do not know vagrant, as they do not know Terraform (Packet with Terraform) and it leaves the door open to other setup guides. Simplification does not depend on the tools you know or adopt at this point in time, but on the process, itself and the improvements we made on the installation process we made are cross tools or providers. https://github.com/tinkerbell/tink/pulls?q=is%3Apr+is%3Aclosed+setup+author%3Agrahamc 

The coupon is not a big issue for what I can tell, we discussed about the possibility to leave a coupon as part of the Tinkberell Packet setup guide, I think it will happen very soon

Reply:
no, we need both